### ISSUE TITLE
500 Error on Node settings edit

### ISSUE THREAD

**Issue by F0x06** [pterodactyl/panel#5051]
### Current Behavior

500 SERVER ERROR

### Expected Behavior

Working normaly.

### Steps to Reproduce

1. Start provided docker-compose.example.yml file
2. Create admin account with `docker compose exec panel php artisan p:user:make`
3. Log-in
4. Create Location:
     - Short Code: my-location
6. Create Node:
     - Name: test
     - Total Memory: 2048
     - Memory Over-Allocation: 0
     - Total Disk Space: 2048
     - Disk Over-Allocation: 0
     - FQDN: localhost
7. Go to the "Settings" tab of your node and try to change anything, then click "Save Changes"
8. "500 SERVER ERROR"

### Panel Version

1.11.5

### Wings Version

n/a

### Error Logs

```bash
Log files are empty.
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by Boy132** [comment]
This is not a bug, please visit the Discord for configuration issues.

**Comment by F0x06** [comment]
> This is not a bug, please visit the Discord for configuration issues.

Thank you for the response. I understand that this might be stemming from a configuration issue. In my experience, unhandled 500 errors often point towards areas where error handling could potentially be enhanced.

Could you provide more detailed information on common causes for this error, or any specific configurations that are often overlooked?. 

I encountered this issue when reinstalling my whole setup I had for years. Everything works except node edition, as a workaround I had to change the settings on the database. 

Thanks

**Comment by F0x06** [comment]
Apparently this was caused by a permission error on the log files folder, by default it's owned by `root` when using local path mounts, for example `- "/srv/pterodactyl/logs:/app/storage/logs"`, the issue is not present when using docker volumes. fixed by using this command `docker compose exec panel chown -R nginx: /app/storage/logs/`, found on discord https://discord.com/channels/122900397965705216/493443683685892096/1150639872198070282

Basically if anything gone wrong on the node settings that involves a log message, the error is triggered.

A possible fix could be to ensure correct ownership of `/app/storage/logs/` on the entrypoint script of the container ?

**Comment by TBG-FR** [comment]
> Apparently this was caused by a permission error on the log files folder, by default it's owned by `root` when using local path mounts, for example `- "/srv/pterodactyl/logs:/app/storage/logs"`, the issue is not present when using docker volumes. fixed by using this command `docker compose exec panel chown -R nginx: /app/storage/logs/`, found on discord https://discord.com/channels/122900397965705216/493443683685892096/1150639872198070282
> 
> Basically if anything gone wrong on the node settings that involves a log message, the error is triggered.
> 
> A possible fix could be to ensure correct ownership of `/app/storage/logs/` on the entrypoint script of the container ?

Had this issue today, thanks for the hint ! Now I can reinstall my server without any issue AND I see new log files created under that path (there should maybe be a hint/error/log going "up" so we see it more clearly ? I had nothing new doing `docker logs -f pterodactyl_panel`)

**Comment by Doarlockgg** [comment]
Another possible fix could be commenting this line - "/srv/pterodactyl/logs/:/app/storage/logs" in the compose file
